### PR TITLE
Fix UI layout issues

### PR DESCRIPTION
**Pull request by mrruby**

### CODE REVIEW

File: src/renderer/src/lib/components/AppDetailsPanel.svelte [hunk 1]
```diff
@@ -35,7 +35,9 @@
 				{/if}
 			</div>
 		</div>
-		<slot name="install" />
+		{#if selectedIndex === 0}
```
Comment (matthme) [inline src/renderer/src/lib/components/AppDetailsPanel.svelte]: What's the reason for this change? It seems unintuitive for me that the button disappears when I switch tab. In the top panel It also displays the latest version number so I think it is clear enough that this install button refers to installing the latest version...

Comment (mrruby) [reply src/renderer/src/lib/components/AppDetailsPanel.svelte]: We talked about this yesterday, but I'm not strongly in favour for taking it out in this case. I'll undo this change and we can review it again on our next call.

**Review by matthme — APPROVED** [review]

File: src/renderer/src/lib/components/AppDetailsPanel.svelte (hunk 1 above)
Comment (matthme) [reply src/renderer/src/lib/components/AppDetailsPanel.svelte]: Okay. I understood from the call that the button only gets disabled (i.e. greyed out, not removed) whenever one of the buttons is loading after having been pressed.
